MdeModulePkg/Universal/SetupBrowserDxe/IfrParse.c

```
@@ -1551,34 +1638,43 @@ ParseOpCodes (
       //
       if ((CurrentExpression == NULL) && (MapScopeDepth > 0)) {
         CurrentExpression = CreateExpression (CurrentForm, OpCodeData);
+        if (CurrentExpression == NULL) {

```

I don't think ignoring errors is the right action here.
Throughout this ridiculously long function, please break or return instead.

---

This is outside of the switch statements, so break won't work. Its still in the while loop, so a continue could work, but that can lead to more weird behavior.